<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Units</string>
  </property>
  <property name="windowIcon">
   <iconset resource="units.qrc">
    <normaloff>:/icons/equals.png</normaloff>:/icons/equals.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDistance"/>
   <addaction name="actionWeight"/>
   <addaction name="actionTemperature"/>
   <addaction name="separator"/>
   <addaction name="actionClear"/>
   <addaction name="actionAbout"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="units.qrc">
     <normaloff>:/icons/fallback/actions/edit-clear.png</normaloff>:/icons/fallback/actions/edit-clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear Fields</string>
   </property>
   <property name="toolTip">
    <string>Clear All Fields</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="units.qrc">
     <normaloff>:/icons/fallback/actions/application-exit.png</normaloff>:/icons/fallback/actions/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit Program</string>
   </property>
   <property name="shortcut">
    <string>Alt+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="units.qrc">
     <normaloff>:/icons/fallback/actions/help-about.png</normaloff>:/icons/fallback/actions/help-about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDistance">
   <property name="icon">
    <iconset resource="units.qrc">
     <normaloff>:/icons/fallback/actions/length-48.png</normaloff>:/icons/fallback/actions/length-48.png</iconset>
   </property>
   <property name="text">
    <string>Distance</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionTemperature">
   <property name="icon">
    <iconset resource="units.qrc">
     <normaloff>:/icons/fallback/actions/temperature-48.png</normaloff>:/icons/fallback/actions/temperature-48.png</iconset>
   </property>
   <property name="text">
    <string>Temperature</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionWeight">
   <property name="icon">
    <iconset resource="units.qrc">
     <normaloff>:/icons/fallback/actions/weight-48.png</normaloff>:/icons/fallback/actions/weight-48.png</iconset>
   </property>
   <property name="text">
    <string>Weight</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="units.qrc"/>
 </resources>
 <connections/>
</ui>
